rbac/management/workspace/model.py

```diff
@@ -55,6 +56,11 @@ def save(self, *args, **kwargs):
         self.full_clean()
         super().save(*args, **kwargs)
 
+    def clean(self):
+        """Validate the model."""
+        if self.type != self.Types.ROOT and self.parent_id is None:
+            raise ValidationError({"parent_id": ("This field cannot be blank.")})
```

Comment:
```suggestion
            raise ValidationError({"parent_id": ("This field cannot be blank for non-Root-type Workspaces.")})
```
(or something like that)

Reply:
I'm happy to make the change to be more explicit. I was going for consistency here since this emulates the validation error raised by the API on required fields. That said, the clients are different (end user vs internal caller).